entrypoint.sh

```diff
@@ -16,7 +16,7 @@ echo "Ensuring our base configuration is present in the database"
 
 if [ -v SENTRY_BACKEND_DSN ]; then
     echo "Testing Sentry configuration"
-    echo "from sentry_sdk import capture_message;capture_message('This is a test event')\n" | ./manage.py shell
```

Comment:
This seems like it might be approaching an argument for turning this into a management command which can be linted and tested more easily

Reply:
I was kind of hoping that feature would make its way into the new sentry SDK eventually.

Reply:
I ticketed that for future work